cmd/version/manager.go

```diff
@@ -0,0 +1,163 @@
+package version
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/charmbracelet/log"
+	gover "github.com/hashicorp/go-version"
+)
+
+var (
+	managerInstance     Manager   //nolint:gochecknoglobals
+	managerInstanceOnce sync.Once //nolint:gochecknoglobals
+)
+
+// InitManager initializes the version manager, we have an explicit init so we can capture the root
+// context to ensure appropriate cancellation during the version fetching if necessary.
+func InitManager(
```

Comment:
this dude gets called from the pre exec func so it should be always available to the cli -- and since it shouldn't ever matter to a future http-able clab this should be chill here (I think)